internal/cmd/login.go

```diff
@@ -218,12 +239,8 @@ func loginToInfra(cli *CLI, lc loginClient, loginReq *api.LoginRequest, noAgent
 	if err != nil {
 		if api.ErrorStatusCode(err) == http.StatusUnauthorized || api.ErrorStatusCode(err) == http.StatusNotFound {
 			switch {
-			case loginReq.AccessKey != "":
-				return &LoginError{Message: "your access key may be invalid"}
```

Comment:
Should this be removed? Access key looks unchanged so removing it may lead to some confusing errors

Reply:
Thanks, I must have incorrectly committed this